ci_framework/roles/tempest/tasks/configure-tempest.yml

```diff
@@ -0,0 +1,32 @@
+- name: Get keystone secret name
+  register: keystone_secret_name
+  environment:
+    PATH: "{{ cifmw_path }}"
+  ansible.builtin.shell:
+    cmd: oc get keystoneapi keystone -o json | jq -r .spec.secret
```

Comment:
It would be better to explicitly specify the OCP namespace, where the command needs to run, to avoid any errors if the active namespace gets switched due to external factors during the execution.
Eg: https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/roles/artifacts/tasks/cluster_info.yml#L38